pkg/pgmodel/new_migrate.go

```diff
@@ -23,7 +23,7 @@ var (
 )
 
 func doesSchemaMigrationTableExist(db *pgx.Conn) (exists bool, err error) {
-	const stmt = "SELECT count(*) FILTER (WHERE tablename = 'prom_schema_migrations') > 0 FROM pg_tables"
+	const stmt = "SELECT count(*) FILTER (WHERE schemaname = 'public' AND tablename = 'prom_schema_migrations') > 0 FROM pg_tables"
```

Comment:
Any reason to use `filter` over simple `where` clause? Imo it's less readable like this